### ISSUE TITLE
We need more reasons to use laser guns

### ISSUE THREAD

**Issue by Kayek14** [space-wizards/space-station-14#18280]
I think the game could be improved if we saw more laser guns being used. Right now the only time we really see them is with salvage or the captain's antique laser. Most of the time they are just... non-existent and are so heavily outperformed by ballistic guns that no one really bothers with using them.

Laser fights are also so much more enjoyable. Each shot seems to have more significance, fights last longer and offers more opportunities for tactics. Currently, most gun fights are with ballistic guns and you are dead in 1-2 seconds.

Laser weapons need some love, as of now they are shit compared to any ballistic gun. their TTK in some cases double or triple that of ballistic guns, and their ammo capacity is usually about 1/3 of them as well. Getting into a fight with a laser gun when your opponent has a regular boolet shooting gun is almost always a loss. Their damage type also cauterizes bleeding so that further hampers their possible damage output.

Of course laser guns have one strong advantage, they can shoot through glass. 

I don't think this one factor should be making them so limited in gunfights. Yes, they should always be out performed by ballistic weapons in most cases, but I think the game would be improved if we buff laser guns slightly. Increasing their battery size seems like the easiest option to me right now.

Traditionally in ss13, most of the guns available to the station were lasers. Syndicate/traitors/nukies were the ones with ballistic guns. That was why redsuits had high heat resistance and secoffs had armor vests. However i understand that 14 is a different game with our own game philosophy. 

That being said, as of now in the current state of the game, 90% of all gunplay we see is with ballistic weaponry. Perhaps make laser guns more easily obtainable early game, and make ballistic guns late game? or split the guns in the armory to be 50% laser and 50% ballistic?

TLDR: Ballistic guns outperform laser guns way too much. Everyone uses ballistic guns and you are dead in 1-2 seconds. Introduce more reasons to use laser guns or make ballistic guns more rare.

**Comment by LankLTE** [comment]
> TLDR: Ballistic guns outperform laser guns way too much. Everyone uses ballistic guns and you are dead in 1-2 seconds. Introduce more reasons to use laser guns or make ballistic guns more rare.

To say that nobody uses them is an overstatement. Lasers already have the ability to shoot through windows and are hitscan, which will inevitably make them have some niche. 
The truth is their DPS is lower as is their total damage, so they'll almost always be second to ballistic weapons in an open fight. But I don't think that's necessarily a problem. I'm not going to go through and name every scenario in which a laser rifle is better, but there is a lot, and they do those roles well enough to not need changes. There's a reason everyone orders lasers instead of SMGs when nukies show up.

**Comment by Aisu9** [comment]
most of the laser weapon target Heat resistance, which is a pretty good niche in itself i think

**Comment by Aisu9** [comment]
Well... Laser guns suffer from not being able to change battery (which may be the main downside, and probably one of the hardest one compared to every weapons in the game).
So yeah, if you were able to switch the battery in with another that you have in your pocket, laser weapon would be WAY MORE usable instead of a single clip hitscan gun

**Comment by Eden077** [comment]
> Well... Laser guns suffer from not being able to change battery (which may be the main downside, and probably one of the hardest one compared to every weapons in the game).
> So yeah, if you were able to switch the battery in with another that you have in your pocket, laser weapon would be WAY MORE usable instead of a single clip hitscan gun

I think the main issue we'd find with this is someone getting something like the x-ray cannon and an antique powercell, you know cause uh... It'd be pretty busted

**Comment by Aisu9** [comment]
> I think the main issue we'd find with this is someone getting something like the x-ray cannon and an antique powercell, you know cause uh... It'd be pretty busted
Pretty much yeah, but can you even find antique powercells anywhere? i saw it while messing in sandbox but never during salvaging (there could be battery restrictions on many guns to avoid this particular case of OP battery on OP guns)

**Comment by Eden077** [comment]
> > I think the main issue we'd find with this is someone getting something like the x-ray cannon and an antique powercell, you know cause uh... It'd be pretty busted
> Pretty much yeah, but can you even find antique powercells anywhere? i saw it while messing in sandbox but never during salvaging (there could be battery restrictions on many guns to avoid this particular case of OP battery on OP guns)

Nah, did salvage two days ago and one of the missions had an antique, it can spawn it's just rare

**Comment by LankLTE** [comment]
> So yeah, if you were able to switch the battery in with another that you have in your pocket, laser weapon would be WAY MORE usable instead of a single clip hitscan gun

This was removed for a reason

**Comment by fidso** [comment]
> There's a reason everyone orders lasers instead of SMGs when nukies show up.

What? Do you even play the game?

**Comment by LankLTE** [comment]
> What? Do you even play the game?

Shockingly, I do. The last 3-4 nukie rounds I've played people ordered lasers instead of SMGs. Granted a big reason for that is probably how much cheaper they are, so it's not really fair to say that it has anything to do with their performance. Regardless I don't think lasers are in a terrible spot.

**Comment by deltanedas** [comment]
lasers are a better defensive weapon since you can build grilles and windows which both let you fire and have to be broken by attackers using ballistic weapons

**Comment by HoidC** [comment]
lasers are an extremely awful choice against ops because 1. there's usually more than one of them and 2. lasers do awful damage against their very heat resistant armour

people are likely only ordering them because they're cheap - they're by no means good, which I think is fine. imo they should stay cheap and be a crew mass arming weapon

**Comment by GoodWheatley** [comment]
Making an executive decision and closing this, I feel like lasers are in a good place right now. If a maintainer disagrees feel free to reopen.
